cloudapi/insights/client_test.go

```diff
@@ -297,6 +355,96 @@ func TestClient_IngestRequestMetadatasBatch_ReturnsErrorWithFailingServerAndNonC
 	require.ErrorContains(t, err, testErr.Error())
 }
 
+func TestClient_IngestRequestMetadatasBatch_ReturnsErrorWithNoErrorAfterRetrySeveralTimes(t *testing.T) {
```

Comment:
I think the test name here is a bit misleading.

Reply:
I updated to `TestClient_IngestRequestMetadatasBatch_ReturnsNoErrorAfterRetrySeveralTimes`. Thanks!